docs/manage/_advanced.rst

```diff
@@ -249,9 +256,136 @@ Alternatively, you can follow these steps in the GUI.
+
+  .. code-block:: bash
+     
+     kolibri manage benchmark
+
+The command will have an output similar to this:
+
+  .. figure:: img/benchmark.png
+      :alt: Command line output of the 'kolibri manage benchmark' command
+
+      Command line output of the 'kolibri manage benchmark' command
+
+Take a screenshot of the Terminal or Command prompt, or copy and paste the output in the community forum post.
+
+Profile
+^^^^^^^
+
+In order to collect more than a current snapshot of Kolibri server performance, you can use the profiling command. When executed, the command will collect a series of performance indicators every 10 seconds and save them in a CSV file. Type this in the Terminal or Command prompt.
+
+  .. code-block:: bash
+     
+     kolibri manage profile
+
+.. tip:: Command collects and saves the information 60 times by default. If you want to change this value, add the ``--num_samples`` flag with the desired number at the end.
+
+  .. code-block:: bash
+     
+     kolibri manage profile --num_samples=100
+
+
+Each log line contains this information:
+
+* Date and time of each command execution
+* Number of Kolibri active sessions (including guest sessions)
+* Number of Kolibri logged users
+* Number of Kolibri user interactions during the last minute
+* Total percentage of CPU use
+* Total memory use
+* Total available memory
+* Number of processes executed in the server
+* Percentage of CPU used by Kolibri
+* Percentage of memory used by Kolibri
+
+To help us troubleshoot potential problems on your Kolibri server, locate and send us the ``KOLIBRI_HOME/performance/date_time_performance.csv`` file.
+
+
+.. _profile_requests:
+
+
+Profile Server Requests
+"""""""""""""""""""""""
+
+If you have the ``[Server]`` section of the :ref:`OPTION.INI <profile_requests_ini>` file  configured with ``PROFILE = 1``, the above command will additionally perform a profile of every request made by Kolibri server, and save the results in a second log file as ``KOLIBRI_HOME/performance/date_time_requests_performance.csv``
+
+Each log line contains this information:
+
+* Timestamp
+* Request path
+* Time spent processing the request
+* Memory (in KB) used by the Kolibri process when the request came in
+* Memory (in KB) used by the Kolibri process when the response was sent
+* CPU percentage used by the Kolibri process when the request came in
+* CPU percentage used by the Kolibri process when the request was sent
+* Flag indicating if the request is the slowest one since the analysis started
+
+.. warning::
+  Profiling server requests can consume a lot of computer resources, and potentially slow it down. For this reason you need to explicitly allow it in the ``option.ini`` file. Without the ``PROFILE = 1`` key, server requests will not be profiled, and the second CSV file will not be created. 
+
+  
+
+Customize Kolibri Settings with the OPTION.INI File
+---------------------------------------------------
```

Comment:
Might want to put this section higher since it's a prerequisite for other items like `PROFILE = 1` above and `HTTP_PORT = 1234` below